.github/workflows/run-a-single-node-from-branch.yml

```diff
@@ -0,0 +1,156 @@
+name: 'Run a node with selected configuration. NodeName=\"DevNode-${{ github.actor }}-${{ github.ref }}\"'
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: "Select a network on which You want to run a node"
+        default: "mainnet"
+        required: true
+        type: choice
+        options:
+          - mainnet
+          - gnosis
+          - sepolia
+          - goerli
+          - chiado
+      cl_client:
+        description: "Select Consensus Layer Client to run node against"
+        default: ""
+        required: true
+        type: choice
+        options:
+          - lighthouse
+          - lodestar
+          #- nimbus
+          - prysm
+          - teku
+      cl_custom_image:
+        description: "In case of need to run non-default cl image (different than actually supported by Sedge) put it in there"
+        default: ""
+        required: false        
+      config:
+        description: "Select a config file which will be selected for tests."
+        default: "default.json"
+        required: true        
+        type: choice
+        options:
+          - default.json
+          - fastSync.json
+          - fullSync.json
+          - fuzzer.json
+      timeout:
+        description: "Timeout in hours before triggering the deletion of smoke test instances. Maximum time of node lifetime can be 72 hours."
+        default: "24"
+        required: true
+      additional_nethermind_flags:
+        description: "Provide any additional flags to the Nethermind in space-separated format. Example: \"JsonRpc.Enabled=false Sync.SnapSync=false\"."
+        default: ""
+        required: false
+      additional_cl_flags:
+        description: "Provide any additional flags to the CL client in space-separated format. Example: \"clflag1=1 clflag2=2\"."
+        default: ""
+        required: false
+      
+jobs:
+  create_docker_image:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Trigger Docker Build Action
```

Comment:
not sure, but maybe we could use [this action](https://github.com/benc-uk/workflow-dispatch) for triggering to make the yaml a little bit shorter
something like that:

```
- name: Trigger Docker Build Action
- uses: benc-uk/workflow-dispatch@v1
  with:
    workflow: build-nethermind-docker-images
    inputs: '{ "tag": {{ github.ref }}, "dockerfile": "Dockerfile"}'
```

Reply:
I tried a few others but problem was that each had some issues - but didn't tried this one so will take a look :)